backend/services/inventory/tests/tests/conftest.py

```diff
@@ -12,13 +12,33 @@
 #    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
+
 import csv
 
 import pytest
 
 from pymongo import MongoClient
 
-from client import CliClient, ManagementClient, InternalApiClient, ManagementClientV2
+import openapi_client as oas
+
+from client import CliClient
+
+
+@pytest.fixture(scope="session")
+def inventory_attributes(request):
+    attributeList = []
+
+    filename = "inventory_items"
```

Comment:
okey I am getting lost a bit, this file exists where? who creates it?

Reply:
Whelp, can't say I was not surprised too. It used to be a pytest option with this default value, but it was never set anywhere, and since this filename has to be local to the current directory it doesn't make sense to ever change it.
So as a temporary workaround, I just hard-coded the filename for now.